client/src/pages/Register.jsx

```diff
@@ -1,83 +1,202 @@
-                            Username
-                        </label>
-                        <input
-                            className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
-                            id="username"
-                            name="username"
-                            type="text"
-                            placeholder="Username"
-                            value={formData.username}
-                            onChange={handleChange}
-                        />
-                    </div>
-                    <div className="mb-4">
-                        <label className="block text-gray-700 text-sm font-bold mb-2" htmlFor="email">
-                            Email
-                        </label>
-                        <input
-                            className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
-                            id="email"
-                            name="email"
-                            type="email"
-                            placeholder="Email"
-                            value={formData.email}
-                            onChange={handleChange}
-                        />
-                    </div>
-                    <div className="mb-6">
-                        <label className="block text-gray-700 text-sm font-bold mb-2" htmlFor="password">
-                            Password
-                        </label>
-                        <input
-                            className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 mb-3 leading-tight focus:outline-none focus:shadow-outline"
-                            id="password"
-                            name="password"
-                            type="password"
-                            placeholder="******************"
-                            value={formData.password}
-                            onChange={handleChange}
-                        />
-                    </div>
-                    <div className="flex items-center justify-between">
-                        <Button type="submit">
-                            Register
-                        </Button>
+        <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100vh', flexDirection: 'column' }}>
+            <h1 style={{ fontSize: '24px', marginBottom: '20px' }}>New User Registration</h1>
+            <form onSubmit={handleSubmit} style={{ display: 'flex', flexDirection: 'column', width: '300px', gap: '10px' }}>
+
+                <label htmlFor="userName" className={`${isUsernameValid ? 'text-green-700' : 'text-black-700'} block mb-2 text-sm font-medium`}>
+                    Username
+                </label>
+                <input
```

Comment:
can you please use the textinput from flowbite so that we can standardize this especially with the styling? you can refer to the landing page for more info, that way you dont have to do all the stuff manually as well

and for the error messages refer to https://www.flowbite-react.com/docs/components/forms#form-validation

Reply:
same with button etc btw, if you use flowbit it already has the component so its minimal work 
client/src/pages/LandingPage.jsx check here for more info!

Reply:
Oops, my bad on this - I was pulling code from flowbite - not flowbite-react. I will adjust in a future PR